### ISSUE TITLE
Publications that don't meet accessibility requirements

### ISSUE THREAD

**Issue by mattgarrish** [mattgarrish/epub-revision#2569]
The need to differentiate a publication that hasn't been checked for accessibility versus one that doesn't meet the EPUB accessibility standard came up again while trying to work out the display metadata in the CG's accessibility task force editors call.

This sounds like it will be important to know in EU for regulatory purposes, so parsing a summary for claims isn't optimal. @gregoriopellegrino has also indicated that ONIX is updating the "inaccessible" code point value to reflect that such books aren't necessarily inaccessible to everyone (the definition will also apply to books with "known accessibility limitations").

The idea from the call is to recommend/note that authors use the conformance value "none" to indicate this:

```
<meta property="dcterms:conformsTo">none</meta>
```

This could be a requirement for publishing in the EU.

**Comment by iherman** [comment]
Does this proposal include a proposed change in the EPUB 3.3 specification? Or in any other publication of the EPUB family?

**Comment by shiestyle** [comment]
This will be a small change for EPUB-A11Y 1.1 in the EPUB 3.3 specification.

"dcterms:conformsTo" doesn't look to be mandatory now and should we set it MUST if we prepare a value of "none"?
If so, it will not be a small change...

**Comment by mattgarrish** [comment]
> "dcterms:conformsTo" doesn't look to be mandatory now

It is a mandatory reporting component if you meet the requirements of the standard, so this is not a new feature request. It'd be a class 3 clarification, at most.

> and should we set it MUST if we prepare a value of "none"

This is unlikely. We can't make it a requirement to say that you fail the specification's requirements. It's not enforceable without making conformance part of the core specification.

We can recommend using the value "none" in [s. 3.5.2](https://www.w3.org/TR/epub-a11y-11/#sec-conf-reporting-pub), but in a non-binding way. For example:

> Although EPUB creators can report that an EPUB publication does not conform to this standard, and is not an optimized publication, in the accessibility summary, a more explicit indicator may be preferred. In these cases, it is RECOMMENDED to set a `dcterms:conformsTo` property with value "`none`".

We could then perhaps note that this is expected to be done in the EU in the mapping document. Does that work for you @gregoriopellegrino?

**Comment by gregoriopellegrino** [comment]
I think it may be useful to express two pieces of information, slightly different:

- I have not tested the accessibility of the publication and so I know nothing about it
- I have tested the accessibility of the publication and know that there are problems with it

Is there any way to represent both of these pieces of information?

**Comment by iherman** [comment]
Thanks for the responses. I have set the labels to make it clear it affects the Accessibility spec and I have also set the EU mapping spec (note). 

Based on what I read, the modification would be [class-3](https://www.w3.org/2023/Process-20230612/#class-3) changes. I have set the label accordingly, but change, please, if I misunderstood...

**Comment by shiestyle** [comment]
@mattgarrish I recognized that 3 metadata like "accessMode" are mandatory but "dcterms:conformsTo" and "a11y:certifiedBy" are required only when adopting to WCAG. So I just thought that it's simple if all 5 metadata are required but it's not my strong request. Class 3 change and recommendation will be fine.

**Comment by mattgarrish** [comment]
> I recognized that 3 metadata like "accessMode" are mandatory but "dcterms:conformsTo" and "a11y:certifiedBy" are required only when adopting to WCAG.

They're two completely different areas of the specification, yes. You have to meet the requirements of the discovery section to conform to the standard, but you also have to include a `conformsTo` property to state you meet all the requirements.

My point is only to clarify that a requirement already exists (this is not a new feature) and the proposed change only represents a minor (non-mandatory) clarification, so if reviewers check this issue later it's clear that this is only going to be a class 3 change.

**Comment by mattgarrish** [comment]
> I have not tested the accessibility of the publication and so I know nothing about it

Isn't this the same as not finding a `conformsTo` property?

**Comment by gregoriopellegrino** [comment]
> > I have not tested the accessibility of the publication and so I know nothing about it
> 
> Isn't this the same as not finding a `conformsTo` property?

Ok, fine.

**Comment by mattgarrish** [comment]
We could consider a value like `unknown`, but I'd be concerned that we'd be unintentionally endorsing people not checking their publications.

I could see using a placeholder like this in an internal workflow prior to checking the publication, but I'm not convinced we'd see publishers not checking their publications taking it up.

**Comment by gregoriopellegrino** [comment]
For me is fine the absence of `conformsTo` to express unkown, we only need to provide guidance to content producers.
